scipy/stats/__init__.py

```diff
@@ -377,6 +377,9 @@
 
    rvs_ratio_uniforms
    NumericalInverseHermite
+   TransformedDensityRejection
+   DiscreteAliasUrn
+   UNURANError
```

Comment:
Is this something that users should be able to instantiate? See discussion in gh-13462.

Reply:
> If it's really important to document this separately, consider putting it in [Warnings used in `scipy.stats`](https://34623-1460385-gh.circle-artifacts.com/0/html-scipyorg/reference/stats.html#warnings-used-in-scipy-stats).

I think it is better to document it in case the user is surprised to see a `UNURANError`. I will move this in the `Warnings used by scipy.stats` section.

Reply:
I agree that it should be documented. My point was just that "documenting it" does not necessarily require that it have its own link directly from the top-level documentation.

Reply:
Done.